client/pages/room.jsx

```diff
@@ -28,6 +29,19 @@ export default class Room extends React.Component {
   }
 
   componentDidMount() {
+
+    this.socket = io('/room', {
+      query: {
+        roomId: this.props.roomId
```

Comment:
Be sure to disconnect this socket when the component unmounts so you don't leak connections

```js
componentWillUnmount() {
  this.socket.disconnect();
}
```

https://reactjs.org/docs/react-component.html#componentwillunmount